app/src/main/java/com/arkbuilders/arkdrop/presentation/feature/qrcodescanner/QRCodeScannerViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.arkbuilders.arkdrop.presentation.feature.qrcodescanner
+
+import android.content.Intent
+import android.graphics.Rect
+import android.net.Uri
+import android.view.MotionEvent
+import android.view.View
+import com.google.mlkit.vision.barcode.common.Barcode
+
+class QRCodeScannerViewModel(barcode: Barcode) {
+
+    var boundingRect: Rect = barcode.boundingBox!!
```

Comment:
Could we handle unrecognizable QR code more elegantly, instead of crashing the app with `barcode.boundingBox!!` here?

`barcode.boundingBox!!` could return `null` if bounding box of the QR code cannot be determined.